aio/content/marketing/contributors.json

```
@@ -667,6 +667,13 @@
     "groups": ["Angular"],
     "lead": "dennispbrown"
   },
+  "rockument69": {
+    "name": "Tony Bove",
+    "picture": "tonybove.jpg",
+    "bio": "Tony is a technical writer with Expert Support. His lifelong passions are helping people use technology, writing fiction, and playing music. When he's not working or playing the harmonica with friends in a bluegrass band, he's swimming and snorkeling on a Kauai beach and playing ball with his Irish Wolfhound. He's worked at home for decades before it became a thing.",
+    "groups": ["Angular"],
+    "lead": "dennispbrown"
```

My understanding that your technical lead is @aikidave (lead here is meant "TL" and not the "manager")
```suggestion
    "lead": "aikidave"
```

---

OK, included in the following commit.

---

Thanks for the welcome! Happy to be on board.